tests/adapters/compliance_tests/add_bond.py

```diff
@@ -0,0 +1,28 @@
+from hamcrest import is_, assert_that
+from netman.core.objects.exceptions import BondAlreadyExist, BadBondNumber
+from tests.adapters.compliance_test_case import ComplianceTestCase
+
+
+class AddBondTest(ComplianceTestCase):
+    _dev_sample = "juniper"
+
+    def test_creates_the_bond(self):
+        self.client.add_bond(42)
+
+        bond_from_get_bond = self.client.get_bond(42)
+
+        assert_that(bond_from_get_bond.number, is_(42))
+
+        self.janitor.remove_bond(42)
+
+    def test_raises_if_create_bond_twice(self):
```

Comment:
"raises if bond already exists".     some other system could have created the bond the firs time to this is not correct